styles/find-and-replace.less

```diff
@@ -166,6 +166,25 @@ atom-workspace.find-visible {
       margin-right: @component-padding;
     }
   }
+
+  .wrap-icon {
+    @wrap-size: 96px;
+
+    display: none;
+    position: absolute;
+    left: 50%;
+    top: @wrap-size * -1.5;
```

Comment:
I just prefer it to be closer to the find view and not directly in the users face. Since search results (at least when they are isolated) are often shown in the middle of the editor the icon view would block the user from seeing the actual result when wrapping, and thereby having to wait for the icon view to fade out.

I guess it's a matter of taste :)